examples/lines_bars_and_markers/stairs_demo.py

```diff
@@ -0,0 +1,60 @@
+"""
+=============
+Histline Demo
+=============
+
+This example demonstrates the use of `~.matplotlib.pyplot.stairs`
```

Comment:
same here

Reply:
I think it would be helpful to point out here that edges has one more element than values, and point out the difference, explicitly, with `step`.